kubernetes/loculus/silo_import_wrapper.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
+lastSnapshot=0
 while true
 do
-    bash /silo_import_job.sh
+    lastSnapshot=$(bash /silo_import_job.sh --lastSnapshot=$lastSnapshot)
```

Comment:
Ah I see to use the script output. We can also just write the snapshot value to a file. And read from the file.

Reply:
Yes actually lets do that - much easier to handle edge cases